contributors/design-proposals/storage/csi-migration.md

```diff
@@ -287,8 +287,22 @@ existing Pods in the ADC.
 
 
 ### Volume Resize
-
-TODO: Design
+#### Offline Resizing
```

Comment:
can you add the details of the mechanism we will use to handoff/synchronize between the `GenerateExpandVolumeFunc` and the `external-reisizer`.

Reply:
Updated.